### DISCUSSION TITLE
Installation on the docker

### DISCUSSION THREAD

**Discussion by gkeb** [music-assistant/support#1187] [Q&A] [answered]
I have a slightly different situation. HA installation is on separate docker. I don't have a supervisor, so I don't have add-ons. I have the whole installation set up on a NAS (Synology). How do I put it up properly here so that this latest version will work? Beta?

**Comment by OzGav** [comment, marked as answer]
Please refer to the third post here https://github.com/music-assistant/hass-music-assistant/issues/1143

**Reply by gkeb** [reply]
It helped. I installed on Synology with such a command, maybe it will be useful for someone in the future:
`docker run --name="music-assistant-server" --net=host --restart=unless-stopped --privileged -v /volume1/docker/music-assistant-server/data:/data ghcr.io/music-assistant/server`
I have locally available under my_ip:8095. just how to add it to Home Assistant now? To make it visible and controllable there too? Because it's not there.

**Reply by OzGav** [reply]
The HA integration is not yet written. Expect it in a couple of weeks

**Reply by tcs2tx** [reply]
I added it to my HomeAssistant dashboard by using an iframe pantel - https://www.home-assistant.io/integrations/panel_iframe/.
